mk/PcapPlusPlus.mk.common

```diff
@@ -24,3 +24,5 @@ CXXFLAGS := -O2 -g -Wall
 endif
 
 PCAPPP_BUILD_FLAGS += $(CXXFLAGS)
+
+BUILD_FLAGS := $(PCAPPP_BUILD_FLAGS)

```

Comment:
Let's just revert the change and keep it as it was?

Reply:
Which version do you mean? Keep PCAPPP_BUILD_FLAGS as it was and extend PCAPPP_INCLUDES with the output of
"pkg-config --cflags" ?

Reply:
We can remove `BUILD_FLAGS := $(PCAPPP_BUILD_FLAGS)` from this file and use `$(PCAPPP_BUILD_FLAGS)` in `Common++/Makefile` and `Packet++/Makefile` (which is the same as we currently have in `dev` and `master` branches).

I suggest we focus this PR on making DPDK build work with `pkg-config`. We can leave additional improvements to a separate PR. Please let me know what you think

Reply:
I agree, pushed a new version